views/cart.html

```diff
@@ -0,0 +1,279 @@
+                                add
+                            </span>
+                        </button>
+    
+                        <input type="number" class="input input-qty" value="2" min="1">
+                        <button class="btn-outline-primary btn pl-04 pr-04 pt-03">
+                            <span class="material-icons icon-menu">
+                                remove
+                            </span>
+                        </button>
+                    </div>
+                    <button class="btn-primary btn btn-py-1 flex flex-justify-center width-full ">
+                        <span class="icon-default-pl">Remove from cart</span>
+                    </button>
+
+                    
+                </div>
+
+                
+            </div>
+
+            <div class="ui-card-list card-py gap shadow-sm mb-1">
+                
+                <div class="ui-card-list-img relative">
+                    <img src="https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_508,h_320,c_fill/gfq5cpqjkllobns88bg8"
+                        alt="chinese work" loading="lazy">
+                    <div class="card-badge">
+                        <span>Pro- Get 20% off</span>
+                    </div>
+                </div>
+                <div class="ui-card-list-content">
+                    <span class="font-bold ">Chicken Personal Wok Bowl</span>
+                    <div>
+                        <span>&#x20b9;</span>
+                        <span class="font-md-light">294</span>
+                    </div>
+                    <span class="font-md-light">Light eater - choice of any 1 rice/noodles, 1 sauce, 4
+                        vegetables and 2 seasoning.</span>
+                </div>
+                <div class="ui-card-list-button gap flex flex-column">
+                    <div class="flex flex-row gap">
+                        <button class="btn-outline-primary btn pl-04 pr-04 pt-03">
+                            <span class="material-icons icon-menu">
+                                add
+                            </span>
+                        </button>
+    
+                        <input type="number" class="input input-qty" value="4" min="1">
+                        <button class="btn-outline-primary btn pl-04 pr-04 pt-03">
+                            <span class="material-icons icon-menu">
+                                remove
+                            </span>
+                        </button>
+                    </div>
+                    <button class="btn-primary btn btn-py-1 flex flex-justify-center width-full ">
+                        <span class="icon-default-pl">Remove from cart</span>
+                    </button>
+
+                    
+                </div>
+
+                
+            </div>
+           </div>
+
+
+            
+
+
+
+           
+          
+            
+
+
+
+
```

Comment:
Suggestion- Remove extra space. If you want to diffrentiate between different components use comments.

Reply:
done